### ISSUE TITLE
Bug OnConfirmed v2.2.2

### ISSUE THREAD

**Issue by Fred7877** [jantinnerezo/livewire-alert#63]
Hi @jantinnerezo !
I have update livewire-alert on 2.2.2 version and i get 2 errors.

The context : 
I have a list of elements in a table, and each element can be deleted with a button which is a livewire component.
When this button is clicked, a $emitSelf('remove') event is fired.

I have the implemented the remove method like that : 

```
public function remove()
    {
        $this->confirm($this->messageModalAsk, [
            'onConfirmed' => [
                'component' => 'self',
                'listener' => 'confirmed'
            ],
            'confirmButtonText' => 'Oui',
            'cancelButtonText' => 'Non',
        ]);
    }

```
And I had this error in the chrome console : 

**Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '$wire')**

I have inspected the LivewireAlert.php file and at the line 111, there is : 

`Arr::set($event, 'id', null);`

I have changed for this : 

`Arr::set($event, 'id', $this->id);`

Because id property is used in livewire-alert.js 
```
Livewire.find(interaction.onConfirmed.id)
                .emitSelf(interaction.onConfirmed.listener, interaction.result);
```

and that clear this error.

Second error,
Now that the confirmed method can only be called by its own component (and that's really cool), I implemented it like this : 
```
 public function confirmed()
    {
        $model = $this->modelType::find($this->modelId);
        $model->delete();
        $this->flash('success', $this->messageModalSuccess, [], $this->routeRedirect);
    }
```

The flash message appears well, but i get an error after in the chrome console: 

**Uncaught (in promise) ReferenceError: data is not defined
    at window.flashAlert (livewire-alert.js:47)**

Regards,

**Comment by jantinnerezo** [comment]
Hi @Fred7877 

You don't need to set `component` to self anymore since it'll get the ID by default. You only set it when you want to emit the event to specific component. 

For the flash issue I'll look into today.

**Comment by Fred7877** [comment]
ah yes ! of course !!! No need to set component to self !!! This is why getEventProperties() return the id and component => 'self' !

Thanks ! 

I'll leave the issue open for the flash issue if want answer it?

**Comment by jantinnerezo** [comment]
Fixed on hotfix release [2.2.3](https://github.com/jantinnerezo/livewire-alert/releases/tag/2.2.3)

Thank you for reporting the issue @Fred7877
